distribution/lib/Standard/Base/0.0.0-dev/src/Network/Email/Email_Address.enso

```diff
@@ -0,0 +1,32 @@
+import project.Data.Text.Text
+import project.Errors.Common.Missing_Argument
+import project.Metadata.Display
+import project.Metadata.Widget
+from project.Metadata.Choice import Option
+from project.Metadata.Widget import Text_Input, Vector_Editor, Single_Choice
+
+## The email address of a sender or recipient.
+type Email_Address
+    ## An email address with an optional display name.
+    Address address:Text=(Missing_Argument.throw "address") name:Text=""
+
+    ## ---
+       private: true
+       ---
+    to_text self -> Text = 
+        if self.name == "" then self.address else self.name + ' <' + self.address + '>'
+
+    ## ---
+       private: true
+       ---
+    default_widget (display : Display = ..Always) -> Widget =
```

Comment:
- Can these widget methods (`default_widget`, `vector_widget`) be made `private`?
- Or even moved outside of API type into some `private` module?

Reply:
- GUI isn't calling `default_widget` directly, is it?
- There is `@from Email_Address.default_widget`
- and GUI just reads `from` annotation which then calls the `default_widget`
- please move the methods into a `private` module
- should something be broken, please let me know

Reply:
Because it will touch 10s of files that are completely unrelated to this MR. And I'm 70% sure it will break something, so if it does I want it isolated in a single commit.